vtex/toolbelt
# Fix new login flow

#### What is the purpose of this pull request?
There were two issues with the new login flow, this PR solves them:
- For some reason the previous package used to get a free port wasn't working properly on some OSs - The package was then changed from `get-port` to `detect-port`.
- In WSL2 the address `127.0.0.1:port` wasn't reachable. This was solved by specifying the hostname on `server.listen`.

Even though changing the address solved the problem for WSL2, some older WSL2 versions will still have this issue, and the solution is update to Windows to the latest version (at least 19041.421). However a reliable way of detecting if the user is on WSL2 or WSL1 is not that easy and because of this as a quick fix I decided to warn all WSL users about possible login issues regarding outdated windows versions.

#### How should this be manually tested?
```
yarn global add vtex@2.107.1-beta.1
```
Check if `vtex login` works properly.

#### Types of changes

#### Chores checklist
